tests/sensors/test_base.py

```diff
@@ -1022,31 +1008,20 @@ def test_sensor_with_xcom_fails(self, make_sensor):
         assert actual_xcom_value is None
 
     @pytest.mark.parametrize(
-        "executor_cls_mode",
+        "executor_cls, expected_mode",
```

Comment:
Hm. Also here it's strange that you remove those -> the tests should have "providers configuration loaded" so they should all work fine.

Reply:
So, I decided to move away from the "test everything in the Airflow ecosystem" approach, since this test is really about making sure `change_sensor_mode_to_reschedule` is honored.

And we don't even get here now, the CE import up top is where this blows up.

Reply:
Sure.